src/content/app/species/components/species-title-area/SpeciesTitleArea.module.css

```diff
@@ -40,6 +40,26 @@
   line-height: 1;
 }
 
+.assemblyCountWrapper {
+  display: inline-flex;
+  grid-area: assembly-count;
+  column-gap: 8px;
+}
+
+.assemblyCountLozenge {
+  background: var(--color-black);
+  border-radius: 10px;
+  color: var(--color-white);
+  font-weight: var(--font-weight-bold);
+  margin-left: 5px;
+  padding: 0 8px;
+}
```

Comment:
We have a dedicated InfoPill component; this class should be either smaller or unnecessary

Reply:
Removed this class completely ✅